### PR TITLE
BTO-690 - adding-flows-test-image

### PR DESCRIPTION
**Pull request by jaymberry**
## Description

This is an image for testing flows.

## Jira link(s)
- https://opennms.atlassian.net/browse/BTO-690

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

**Comment by agalue** [comment]
Out of curiosity: why add the `udpgen` binary file if it is not used? The `Dockerfile` retrieves that binary from its Docker Image in the first phase.

File: test-automation/flows-test-image/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# snmpd-udpgen
+It is a image for generate flow data while it still can receive snmp. It is valicating docker philosophy. However, it is for testing purpose.
```
Comment (jbeuree) [inline test-automation/flows-test-image/README.md:2]: SP? (Valicating)

File: test-automation/flows-test-image/bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  while getopts "u:,s:" o; do
+      case "${o}" in
+          s)
+              SNMP=${OPTARG}
+              ;;
+          u)
+              UDPGEN=${OPTARG}
+              ;;
+          *)
+              echo "-u 'udpgne args' -s 'snmp args'"
```
Comment (jbeuree) [inline test-automation/flows-test-image/bootstrap.sh:20]: udpgen?

**Review by jbeuree — APPROVED** [review]
Looks good, just a couple very minor typos.

**Comment by jaymberry** [comment]
> Out of curiosity: why add the `udpgen` binary file if it is not used? The `Dockerfile` retrieves that binary from its Docker Image in the first phase.

Yes, that was from an old round of development, my mistake, has been removed.

File: test-automation/flows-test-image/README.md (hunk 1 above)
Comment (jaymberry) [reply test-automation/flows-test-image/README.md:2]: Taken from @freddy61025 's README.md, will review.

File: test-automation/flows-test-image/bootstrap.sh (hunk 2 above)
Comment (jaymberry) [reply test-automation/flows-test-image/bootstrap.sh:20]: Yes, will fix that.

**Pull request merged**